tests/acceptance/features/apiAntivirus/antivirus.feature

```diff
@@ -84,3 +84,46 @@ Feature: antivirus
       | dav-path-version |
       | old              |
       | spaces           |
+
+
+  Scenario Outline: upload a file with the virus to a public share
+    Given using <dav-path-version> DAV path
+    And user "Alice" has created folder "/uploadFolder"
+    And user "Alice" has created a public link share with settings
+      | path        | /uploadFolder            |
+      | name        | sharedlink               |
+      | permissions | change                   |
+      | expireDate  | 2040-01-01T23:59:59+0100 |
+    When user "Alice" uploads file "filesForUpload/filesWithVirus/eicar.com" to "/virusFile.txt" using the WebDAV API
+    Then the HTTP status code should be "201"
+    And user "Alice" should get a notification with subject "Virus found" and message:
+      | message                                                                        |
+      | Virus found in virusFile.txt. Upload not possible. Virus: Win.Test.EICAR_HDB-1 |
+    And as "Alice" file "/uploadFolder/virusFile.txt" should not exist
+    Examples:
+      | dav-path-version |
+      | old              |
+      | new              |
+      | spaces           |
+
+
+  Scenario Outline: upload a file with the virus to a password-protected public share
+    Given using <dav-path-version> DAV path
+    And user "Alice" has created folder "/uploadFolder"
+    And user "Alice" has created a public link share with settings
+      | path        | /uploadFolder            |
+      | name        | sharedlink               |
+      | permissions | change                   |
+      | password    | newpasswd                |
+      | expireDate  | 2040-01-01T23:59:59+0100 |
+    When user "Alice" uploads file "filesForUpload/filesWithVirus/eicar.com" to "/virusFile.txt" using the WebDAV API
+    Then the HTTP status code should be "201"
+    And user "Alice" should get a notification with subject "Virus found" and message:
+      | message                                                                        |
+      | Virus found in virusFile.txt. Upload not possible. Virus: Win.Test.EICAR_HDB-1 |
+    And as "Alice" file "/uploadFolder/virusFile.txt" should not exist
+    Examples:
+      | dav-path-version |
+      | old              |
+      | new              |
+      | spaces           |
```

Comment:
same here

Reply:
There is `@skipOnReva` in this featrure file